README.md

```diff
@@ -30,9 +30,11 @@ The language argument is crucial as certain keywords might be interpreted differ
 The script uses a file called osc-api.json, which represents the OpenAPI specification in JSON format.
 For the Outscale API, the YAML source is converted to JSON using yq.
 
-When generating API calls, COGNAC assumes that the OpenAPI file contains components named CallRequest.
+When generating API calls, COGNAC by default assumes that the OpenAPI file contains components named CallRequest.
 For example, if the API has a call named `CreatePony`, the corresponding component should be located at `#/components/schemas/CreatePonyRequest`.
 
+You can change the suffix of functions using `./configure --function-suffix FUNCTION_SUFFIX`
```

Comment:
You can modify the suffix of functions by using the `--function-suffix=FUNCTION_SUFFIX` option with the `./configure` command.